Packs/AutoFocus/Integrations/FeedAutofocus/FeedAutofocus.py

```diff
@@ -326,55 +326,31 @@ def create_indicators_from_single_sample_response(single_sample):
 
         return indicators
 
-    def get_ip_type(self, indicator):
-        if re.match(ipv4cidrRegex, indicator):
-            return FeedIndicatorType.CIDR
-
-        elif re.match(ipv6cidrRegex, indicator):
-            return FeedIndicatorType.IPv6CIDR
-
-        elif re.match(ipv4Regex, indicator):
-            return FeedIndicatorType.IP
-
-        elif re.match(ipv6Regex, indicator):
-            return FeedIndicatorType.IPv6
-
-        else:
-            return None
-
-    def find_indicator_type(self, indicator):
-        """Infer the type of the indicator.
+    @staticmethod
+    def find_indicator_type(indicator: str) -> str:
+        """
+        Get the type of the indicator.
 
         Args:
-            indicator(str): The indicator whose type we want to check.
+            indicator (str): The indicator whose type we want to check.
 
         Returns:
-            str. The type of the indicator.
+            str: The type of the indicator.
         """
-
-        # trying to catch X.X.X.X:portNum
-        if ':' in indicator and '/' not in indicator:
-            sub_indicator = indicator.split(':', 1)[0]
-            ip_type = self.get_ip_type(sub_indicator)
-            if ip_type:
-                return ip_type
-
-        ip_type = self.get_ip_type(indicator)
-        if ip_type:
-            # catch URLs of type X.X.X.X/path/url or X.X.X.X:portNum/path/url
-            if '/' in indicator and (ip_type not in [FeedIndicatorType.IPv6CIDR, FeedIndicatorType.CIDR]):
+        if ip_type := FeedIndicatorType.ip_to_indicator_type(indicator):
+            # catch URLs of type X.X.X.X/path/url or X.X.X.X:portNum/path/url or X.X.X.X/cidrNumber/path/url
+            # or X.X.X.X:portNum/path
+            if indicator.count('/') > 1 or (
+                ip_type in (FeedIndicatorType.IP, FeedIndicatorType.IPv6) and indicator.count('/') == 1
+            ):
```

Comment:
Not sure I understand this `if`. If the function doesn't return the right type to means there is a bug in the regex no?

Reply:
right now the regex ```urlRegex``` that is defined in the common-server-python is defined only for dns based URLs, for example ```https://github.com/demisto/content/pull/17204``` and not for ```IP``` based URLs such as ```1.1.1.1/path```.  I am not sure if the ```urlRegex``` in common server python should fit to ```IP/path``` so I decided not to change it until the review (I also saw it is being widely used in many places in our code - so don't want break stuff).